Library/Tests/ViewModels/LiveStreamCountdownViewModelTests.swift

```diff
@@ -145,7 +145,7 @@ internal final class LiveStreamCountdownViewModelTests: TestCase {
 
     XCTAssertTrue(self.pushLiveStreamViewControllerLiveStream.lastValue?.isLiveNow ?? false)
     XCTAssertTrue(self.pushLiveStreamViewControllerProject.lastValue?.liveStreams.first?.isLiveNow ?? false)
-    XCTAssertTrue(self.pushLiveStreamViewControllerEvent.lastValue?.stream.liveNow ?? false)
+    XCTAssertTrue(self.pushLiveStreamViewControllerEvent.lastValue?.liveNow ?? false)
```

Comment:
Something that we did in #50 which we could perhaps bring across here is to improve the readability of these:

`XCTAssertEqual(true, self.pushLiveStreamViewControllerEvent.lastValue?.liveNow)`